faf-java-server-app/src/main/java/com/faforever/server/game/GameResult.java

```diff
@@ -0,0 +1,30 @@
+package com.faforever.server.game;
+
+import com.faforever.server.common.ServerMessage;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Set;
+
+/**
+ * Results of a finished game.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class GameResult implements ServerMessage {
```

Comment:
Should be `GameResultMessage`.

I added JavaDoc for `ServerMessage` that explains this.